src/tinypeel/tinypeel.py

```diff
@@ -213,19 +213,166 @@ def generateSingleLocusSegregation(peelingInfo, pedigree, args):
         peelingInfo.segregation[:, :, :] = 0.25
 
 
+def get_probability_options():
+    parse_dictionary = dict()
+    parse_dictionary["geno_error_prob"] = lambda parser: parser.add_argument(
+        "-geno_error_prob",
+        default=0.0001,
+        required=False,
+        type=float,
+        help="Genotyping error rate. Default: 0.0001.",
+    )
+    parse_dictionary["seq_error_prob"] = lambda parser: parser.add_argument(
+        "-seq_error_prob",
+        default=0.001,
+        required=False,
+        type=float,
+        help="Sequencing error rate. Default: 0.001.",
+    )
+    parse_dictionary["recombination"] = lambda parser: parser.add_argument(
+        "-recomb",
```

Comment:
@AprilYUZhang should this be `rec_length`?

Reply:
@AprilYUZhang hmm, the help text is the same as for `rec_length` as is the default suggesting we are confusing something here. At any rate, looking at the table https://github.com/AlphaGenes/AlphaPeel/issues/113 we have `rec_length` as input for the genetic length of the chromosome, `est_rec_prob` to flag estimation of the recombination rate between loci, and `.rec_prob` for file output (I made this clear in https://github.com/AlphaGenes/AlphaPeel/issues/113). There is no `recombination` option that I am aware of, but let me know if I am confusing something.

https://github.com/AlphaGenes/AlphaPeel/issues/97
https://github.com/AlphaGenes/AlphaPeel/issues/104

Reply:
I get your means; I made the mistake that rec_prob is not an option rather than an output file name.